components/templates/Personalizer.js

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
+import Head from 'next/head';
+import ReactMarkdown from 'react-markdown';
+import moment from 'moment';
+import { Oval } from 'svg-loaders-react'
+import { getRecommendations } from '../helpers/api'
+import CustomList from "../molecules/CustomList";
+import DropDown from '../molecules/DropDown';
+import RadioButtons from '../molecules/RadioButtons';
+import Card from "../organisms/Card";
+import AdditionalLink from '../atoms/AdditionalLink';
+import Provinces from '../../data/provinces.json';
+import Months from '../../data/months.json';
+import styles from '../../styles/Personalizer.module.css';
+
+export default function Personalizer(props) {
+
+    const [province, setProvince] = useState(props.t.locale === "en" ? "Ontario" : "Quebec");
+    const [month, setMonth] = useState(moment().format("MMMM"));
+    const [language, setLanguage] = useState(props.t.language);
+    const [loading, setLoading] = useState(false);
+    const [results, setResults] = useState(null);
+    const [errorFlag, setErrorFlag] = useState(false);
+
+    async function getResults() {
+        setLoading(true);
+        try {
+            setResults(await getRecommendations(province, month, language));
+        } catch (error) {
+            setErrorFlag(true);
+            console.log(error);
+        }
+        setLoading(false);
+    }
+
+    return (
+        <div className={styles.container}>
+            <Head>
+                <title>{props.t.meta.title}</title>
+                <meta name="description" content={props.t.meta.description} />
+            </Head>
+            <ReactMarkdown children={props.markdown} className={styles.markdown} components={{
+                ul: CustomList
+            }}/>
+            <div className={styles.form}>
+                <DropDown t={props.t} state={{
+                    get: province,
+                    set: setProvince
+                }} label={props.t.provinceSelectLabel} items={Provinces} id="provinces"/>
+                <DropDown t={props.t} state={{
+                    get: month,
+                    set: setMonth
+                }} label={props.t.monthSelectLabel} items={Months} id="months"/>
+                <RadioButtons t={props.t} state={{
+                    get: language,
+                    set: setLanguage
+                }} label={props.t.languageSelectLabel} id="language" items={[
+                    {
+                        "name": {
+                            "en": "English",
+                            "fr": "Anglais"
+                        },
+                        "short": "English"
```

Comment:
it would be more consistent to use the codes for the shorts, `en`/`fr` no?

Reply:
I thought so to, had set it up that way initially. Later found out the API takes the entire English word only so switched these out.

Reply:
We (Mohamad) build this API right? If so we should change it.